harvard-lts/drs-translation-service
# Remove SSL

**Removes SSL from the container**
* * *

**JIRA Issue**: https://at-harvard.atlassian.net/browse/LTSK8S-562

# How should this be tested?

1. Get the .env from Dee Dee
2. Follow the [README](https://github.com/harvard-lts/drs-translation-service?tab=readme-ov-file#local-setup) instructions for local setup (NOTE: BB version is 2.4.2)
3. Spin it up locally
4. Verify that the healtcheck works: http://localhost:10581/healthcheck

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? no
- integration tests? no
